scripts/bootstrap.sh

```diff
@@ -1,11 +0,0 @@
-#!/bin/bash
-#
-# Bootstrap virtualenv environment and postgres databases locally.
-#
-# NOTE: This script expects to be run from the project root with
-# ./scripts/bootstrap.sh
-
-set -o pipefail
-
-# Install development dependencies
-make requirements-dev npm-install frontend-build
```

Comment:
Curious about this file - it mentions bootstrapping the postgres DB but I can't see any evidence that it ever did that?

Reply:
Maybe it was in the makefile at some point but it certainly doesn't seem useful now.

Reply:
There's nothing in the history. It's not clear to me what it was ever for, I've certainly never used it. Kill it with fire.